core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-code/dot-add-variable/dot-add-variable.component.ts

```diff
@@ -11,7 +13,15 @@ import { DotAddVariableStore } from '@dotcms/app/portlets/dot-containers/dot-con
     providers: [DotAddVariableStore]
 })
 export class DotAddVariableComponent implements OnInit {
-    vm$ = this.store.vm$;
+    vm$ = this.store.vm$.pipe(
+        map((res) => {
+            const variables = res.variables.filter(
+                (variable) => variable.fieldType !== 'Column' && variable.fieldType !== 'Row'
```

Comment:
I wonder if we should do an enum for `Column/Row`

Reply:
Also, can we add a test for this filter?

Reply:
+1